pkg/kube/kube.go

```diff
@@ -149,6 +150,18 @@ func (k *KubeClient) GetApplication(name string) (*wego.Application, error) {
 	return &a, nil
 }
 
+func (k *KubeClient) LabelExistsInCluster(label string) error {
+	cmd := []string{"get", "pods", "--namespace=wego-system", "--show-labels"}
```

Comment:
We shouldn't assume this is all in `wego-system`. Also, it would be nice to filter the output to only contain the labels.